pkg/asset/internal/templates.go

```diff
@@ -806,13 +806,15 @@ data:
     .:53 {
         errors
         health
-        kubernetes cluster.local {{ .ServiceCIDR }} {
+        log . {
+            class error
+        }
+        kubernetes cluster.local in-addr.arpa ip6.arpa {
```

Comment:
Did not follow this. Can you please elaborate a little bit? Not sure why serviceCIDR was needed and is not needed anymore

Reply:
Previously it was only resolving records for service IPs but not pod IPs hence moving it to support all PTR requests

Reply:
Oh, okay. IMHO, it would be nice to be in the commit message (my personal taste is different commits, but that is just me).

Also, just curious: what pod IPs don't resolve? Pod IPs in stateful set?